enclave/core/sgx/calls.c

```diff
@@ -131,6 +134,51 @@ extern bool oe_disable_debug_malloc_check;
 **==============================================================================
 */
 
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+extern const volatile oe_sgx_enclave_properties_t oe_enclave_properties_sgx;
+extern volatile oe_sgx_enclave_properties_t oe_enclave_properties_sgx_mutable;
+extern volatile int _have_eeid;
+
+static oe_result_t _eeid_expand()
```

Comment:
Please make the name more meaningful, e.g. `_eeid_expand_heap`.

Reply:
Done.